src/Evolution/Systems/Cce/OptionTags.hpp

```diff
@@ -130,7 +130,7 @@ struct H5IsBondiData {
       "has been the typical format provided by SpEC simulations. Bondi data is "
       "much more efficient for storage size and performance, but both must be "
       "supported for compatibility with current CCE data sources."};
-  static bool default_value() noexcept { return false; }
+  static bool suggested_value() noexcept { return false; }
```

Comment:
delete?

Reply:
I agree -- I think this one should not have a suggested value anymore (the SXS annex now stores the version that requires a `true`, though non-SpEC codes would have trouble producing that sort of data)